util/shlib_wrap.sh.in

```diff
@@ -1,5 +1,22 @@
 #!/bin/sh
+{-
+    use lib '.';
+    use configdata;
 
+    sub shlib {
+        my $lib = shift;
+        return "" if $disabled{shared};
+        $lib = $unified_info{rename}->{$lib}
+            if defined $unified_info{rename}->{$lib};
```

Comment:
Note that strictly speaking, we don't currently need these rename rules on Unix.  However, should we *ever*...